metadata_backend/helpers/schemas/folders.json

```diff
@@ -22,6 +22,573 @@
+                "title": "Type of contributor"
+              },
+              "nameIdentifiers": {
+                "type": "array",
+                "title": "List of name identifiers",
+                "items": {
+                  "type": "object",
+                  "title": "Name identifier object",
+                  "properties": {
+                    "schemeUri": {
+                      "type": ["string", "null"],
+                      "title": "URI (location) of the name identifier scheme"
+                    },
+                    "nameIdentifier": {
+                      "type": ["string", "null"],
+                      "title": "Location of name identifier"
+                    },
+                    "nameIdentifierScheme": {
+                      "type": ["string", "null"],
+                      "title": "Name of name identifier scheme"
+                    }
+                  }
+                }
+              },
+              "affiliation": {
+                "type": "array",
+                "title": "List of affiliations",
+                "items": {
+                  "type": "object",
+                  "title": "Name affiliation object",
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "title": "Name of the place of affiliation"
+                    },
+                    "schemeUri": {
+                      "type": "string",
+                      "title": "URI (location) of the affiliation scheme"
+                    },
+                    "affiliationIdentifier": {
+                      "type": "string",
+                      "title": "Location of affiliation identifier"
+                    },
+                    "affiliationIdentifierScheme": {
+                      "type": "string",
+                      "title": "Name of affiliation identifier scheme"
+                    }
+                  }
+                }
+              }
+            },
+            "additionalProperties": false
+          },
+          "uniqueItems": true
+        },
+        "dates": {
+          "type": "array",
+          "title": "List of relevant dates to publication",
+          "items": {
+            "type": "object",
+            "title": "Date object",
+            "required": [
+              "date",
+              "dateType"
+            ],
+            "properties": {
+              "date": {
+                "type": "string",
+                "title": "A standard format for a date value"
+              },
+              "dateType": {
+                "type": "string",
+                "title": "Relevance of the date"
+              },
+              "dateInformation": {
+                "type": "string",
+                "title": "Specific event of the date"
+              }
+            },
+            "additionalProperties": false
+          },
+          "uniqueItems": true
+        },
+        "descriptions": {
```

Comment:
don't we collect folder description already, why do we need this ?

Reply:
My understanding of the description property in DOI info is that there might be different types of descriptions eg. Abstract, Table of contents, etc. existing at the same time. Also descriptions might exist in different languages but realistically, there will probably only ever be one description in one language. So yea, not sure if this is needed but I left it in for now as an optional property.

Reply:
we don't have a way to submit forms in multiple language at the moment, so support for multiple languages will come at a different date.

we can leave it for now, however as you mentioned, pretty sure user will not be bother with filling this out.